src/Component.php

```diff
@@ -83,4 +95,13 @@ protected function getConfigDefinitionClass(): string
     {
         return ConfigDefinition::class;
     }
+
+    private function getProjectInfo(Client $client): array
+    {
+        $tokenInfo = $client->verifyToken();
```

Comment:
verify token se volá i v autorizaci, nešlo by to nějak spojit? ale to už jste tuším řešili, že?

Reply:
done